contracts/drops/token/HolographDropsEditionsV1.sol

```diff
@@ -312,6 +316,25 @@ contract HolographDropsEditionsV1 is NonReentrant, ERC721H, IHolographERC721Drop
       });
   }
 
+  function bridgeIn(
```

Comment:
Will there need to be additional logic added in order to support bridging of Drop NFTs still?

Reply:
Enforcer handles all the minting/burning so unless we have some super custom tokenURI logic that is generative and changes on bridge transfers, we do not need anything else.